ompi/mpi/fortran/use-mpi-f08/pmpi-f-interfaces-bind.h

```diff
@@ -98,9 +98,9 @@ end subroutine pompi_buffer_attach_f
 ! Note, too, we don't need a "p" version of the C implementation
 ! function -- Fortran's interfaces provide MPI_ and PMPI_ names for
 ! us; they can just both be bound to the same back-end
-! ompi_buffer_detach_f08 C function.
+! pompi_buffer_detach_f08 C function.
 subroutine pompi_buffer_detach_f(buffer_addr,size,ierror) &
-   BIND(C, name="ompi_buffer_detach_f08")
+   BIND(C, name="pompi_buffer_detach_f08")
```

Comment:
Are you sure about this one?  See the comment directly above this subroutine for an explanation as to why it's `ompi_`, and not `pompi_`.

Specifically: I do not believe that there is a `pompi_buffer_detach_f08` C function.